docs/BreakingChanges/107 Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException.md

```diff
@@ -0,0 +1,35 @@
+## 107: Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException
```

Comment:
{
  "Id": "107",
  "Title": "Accessing a WPF DataGrid's selected items from a handler of the DataGrid's UnloadingRow event can cause a NullReferenceException",
  "Details": "Due to a bug in the .NET Framework 4.5, event handlers for DataGrid events involving the removal of a row can cause a NullReferenceException to be thrown if they access the DataGrid's SelectedItem or SelectedItems properties.",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "E:System.Windows.Controls.DataGrid.UnloadingRow",
    "E:System.Windows.Controls.DataGrid.UnloadingRowDetails",
    "E:System.Windows.Controls.DataGrid.UnloadingRowGroup"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/966623",
  "Notes": "An analyzer would help here because we could check event handlers to see whether SelectedItem is being used. See the bug for more details.\nSource analyzer status: Pri 3",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] E:System.Windows.Controls.DataGrid.UnloadingRow
- [x] E:System.Windows.Controls.DataGrid.UnloadingRowDetails
- [x] E:System.Windows.Controls.DataGrid.UnloadingRowGroup